tests/phpunit/payment-methods/class-wc-stripe-express-checkout-ajax-handler-test.php

```
@@ -98,4 +100,99 @@ public function provide_test_modify_country_locale_for_express_checkout() {
+
+	/**
+	 * Test ajax_add_to_cart sends wp_send_json_error payload on failure.
+	 *
+	 * @runInSeparateProcess
+	 * @preserveGlobalState disabled
+	 */
+	public function test_ajax_add_to_cart_returns_error_for_invalid_product() {
+		Ajax_Test_Helper::init_hooks();
+
+		try {
+			$security_nonce       = wp_create_nonce( 'wc-stripe-add-to-cart' );
```

PHPMD (2.15.0)

[error] The variable $security_nonce is not named in camelCase. (undefined)

(CamelCaseVariableName)